person.rb

```diff
@@ -0,0 +1,42 @@
+class Person
+  attr_accessor :name, :age, :parent_permission
+
+  def initialize(name: 'Unknown', age: 0, parent_permission: true)
+    @id = rand(1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def getter_id
+    @id
+  end
+
+  def getter_name
+    @name
+  end
+
+  def getter_age
+    @age
+  end
+
+  def setter_name(name)
+    @name = name
+  end
+
+  def setter_age(age)
+    @age = age
+  end
```

Comment:
- [ ] good job on creating the `Person` class 👍 you are creating the `setters` and `getters` for `name` and `age` here however you are also setting them at the top of the `Person` class as you can see below. https://github.com/Rustamxon7/OOP-school-library/blob/a5965ca0f74444f3a16af0ebebf873153fff560c/person.rb#L2

Reply:
Thank you for letting me know about these fixable issues! I fixed them step by step. You can check them now. Again, if you spot any missing you are free to mention them! Thank you again!